packages/dbos-cloud/cloudutils.ts

```diff
@@ -96,4 +98,17 @@ export type ValuesOf<T> = T[keyof T];
 
 export function createDirectory(path: string): string | undefined {
   return fs.mkdirSync(path, { recursive: true });
-}
\ No newline at end of file
+}
+
+interface CloudAPIErrorResponse {
+  message: string,
+  statusCode: number,
+  requestID: string,
+}
+
+export function handleAPIErrors(label: string, e: AxiosError) {
+  const logger = getLogger();
+  const resp: CloudAPIErrorResponse = e.response?.data as CloudAPIErrorResponse;
+  logger.error(`${label}: ${resp.message}. Request ID: ${resp.requestID}`);
```

Comment:
Shouldn't the request ID go before the (potentially long) message?

Reply:
If we had long messages it could be more legible. Something resulting in:

`2024-01-25 22:23:27 [error]: [8e43b3d2-90b2-49cd-8dc2-2425c3708682] failed to deploy application dbos-base-test-app: schema migration failed.`

Could be nice. Some more complex solution with Winston could make it appear before the semicolon.